adhocteam/Head-Start-TTADP
# Cm 363 record deployments

**Description of change**
Adds logging deployements to new relic

**How to test**
Force a deploy to sandbox, make sure deploy shows up in new relic.

I tested it and the deploy did show up.

![Screenshot from 2021-03-12 09-36-44](https://user-images.githubusercontent.com/6248903/110954719-bc959f80-8340-11eb-9eb7-b81494fd7276.png)


**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/363

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
